spring-web/src/main/java/org/springframework/web/client/DefaultResponseErrorHandler.java

```diff
@@ -132,9 +154,10 @@ public void handleError(ClientHttpResponse response) throws IOException {
 	 * </pre>
 	 */
 	private String getErrorMessage(
```

Comment:
Ideally, I'd make this method protected - so that I can override it to extract some more specific information from responses that return HTML, XML or some other nonstandard data.

Reply:
Considering something beyond 5.3.x - what about this one? 

I'd love to have an extension point where I can further customize the response body preview - format it right, remove some tags, etc. 

A long time ago, here: https://github.com/spring-projects/spring-framework/pull/1956/files - I even thought about a separate class to extract the body. But a protected method would be just fine.